tekton/src/pipelines/rollback.yml.j2

```diff
@@ -21,6 +21,11 @@ spec:
       type: string
 {% endif %}
 
+    # Cluster config
```

Comment:
Did mean to add this here?  Isn't this needed only for install?

Reply:
This became a requirement because `rollback` pipeline makes use of the `ocp-verify` taskdef:
https://github.com/ibm-mas/cli/blob/andrercm-multiple-fixes/tekton/src/pipelines/rollback.yml.j2#L71

and that sets the `ocp_ingress_tls_secret_name` param:
https://github.com/ibm-mas/cli/blob/andrercm-multiple-fixes/tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify.yml.j2#L20

I guess if we don't plan to make use of that property in other pipelines, i could remove that from the taskdef and just add it as a single entry in the `install` pipeline? Somewhere over here:
https://github.com/ibm-mas/cli/blob/andrercm-multiple-fixes/tekton/src/pipelines/install.yml.j2#L45